theforeman.org/pipelines/release/pipelines/candlepin.groovy

```diff
@@ -8,35 +8,29 @@ pipeline {
         ansiColor('xterm')
     }
 
-    stages {
-        stage('staging-build-repository') {
-            when {
-                expression { candlepin_version == 'nightly' }
-            }
-            steps {
-                git url: "https://github.com/theforeman/theforeman-rel-eng", poll: false
+    environment {
+        PROJECT = 'candlepin'
+    }
 
-                script {
-                    candlepin_distros.each { distro ->
-                        sh "./build_stage_repository candlepin ${candlepin_version} ${distro}"
-                    }
-                }
-            }
-        }
-        stage('staging-copy-repository') {
+    script {
+        env.VERSION = candlepin_version
+    }
```

Comment:
I can't find a good way to set the environment variable as a script. Perhaps this needs to be
```suggestion
    env.VERSION = candlepin_version
```